lib/matplotlib/tests/test_axes.py

```diff
@@ -3072,6 +3072,28 @@ def test_hist_stacked_weighted():
     ax.hist((d1, d2), weights=(w1, w2), histtype="stepfilled", stacked=True)
 
 
+@pytest.mark.parametrize("use_line_collection", [True, False],
+                         ids=['w/ line collection', 'w/o line collection'])
+@image_comparison(baseline_images=['stem'], extensions=['png'], style='mpl20',
+                  remove_text=True)
+def test_stem(use_line_collection):
```

Comment:
Perhaps this may just as well use check_figures_equal to check that the result is the same regardless of `use_line_collection`, and avoid an additional baseline image?

Reply:
I don't think `ax.stem` had an image test before I added this one here, so I think it's maybe worth leaving in.

Reply:
I think it is worth having a visual test here, because it helps a lot when trying to figure out problems (especially with the legend)